### DISCUSSION TITLE
tenv ArchLinux User Repository (AUR) package

### DISCUSSION THREAD

**Discussion by clee231** [tofuutils/tenv#86] [Show and tell]
Just wanted to let you all know, I have published an AUR package for tenv. 

✨[https://aur.archlinux.org/packages/tenv-bin](https://aur.archlinux.org/packages/tenv-bin) ✨ 

It is using the pre-built binaries, hence the `-bin` suffix of the package name. I may consider making a source-compiled package if there is enough demand for it.
This is my first package, so please be kind. I welcome any constructive feedback.

Install the package using yay (the AUR Helper):
```bash
yay tenv-bin
```

You can also install the package manually following the [AUR installing and upgrading packages guide](https://wiki.archlinux.org/title/Arch_User_Repository#Installing_and_upgrading_packages).

**Comment by kvendingoldo** [comment]
Wow! Thanks a lot @clee231. May I ask you to update README.md in tenv?

**Reply by clee231** [reply]
Sure thing! Here's a PR: #87
